distributed/utils_test.py

```diff
@@ -1573,15 +1576,8 @@ def check_thread_leak():
             thread
             for thread in threading.enumerate()
             if thread not in active_threads_start
-            and "Threaded" not in thread.name
-            and "watch message" not in thread.name
-            and "TCP-Executor" not in thread.name
-            # TODO: Make sure profile thread is cleaned up
-            # and remove the line below
-            and "Profile" not in thread.name
-            # asyncio default executor thread pool is not shut down until loop
-            # is shut down
-            and "asyncio_" not in thread.name
+            # FIXME this looks like a genuine leak that needs fixing
+            and "watch message queue" not in thread.name
```

Comment:
If I remove this, two tests fail:
- test_exit_callback
```
E               AssertionError: (<Thread(AsyncProcess SpawnProcess-201 watch message queue, started daemon 140080693032704)>, ['  File "/usr/share/min...le "/usr/share/miniconda3/envs/dask-distributed/lib/python3.7/threading.py", line 296, in wait
E                 \twaiter.acquire()
E                 '])
E               assert False
```
- test_nanny_terminate
```
E               AssertionError: (<Thread(AsyncProcess Dask Worker process (from Nanny) watch message queue, started daemon 140338966554368)>, ['  File...le "/usr/share/miniconda3/envs/dask-distributed/lib/python3.7/threading.py", line 296, in wait
E                 \twaiter.acquire()
E                 '])
E               assert False
```